rgw/v2/tests/s3_swift/test_swift_basic_ops.py

```diff
@@ -260,6 +303,68 @@ def test_exec(config, ssh_con):
                 else:
                     test_info.failed_status("test failed")
 
+        elif config.test_ops.get("new_user", False):
+            log.info("enabling swift versioning")
+            ceph_conf.set_to_ceph_conf(
+                "global", ConfigOpts.rgw_swift_versioning_enabled, "True", ssh_con
+            )
+            log.info("trying to restart services ")
+            srv_restarted = rgw_service.restart(ssh_con)
+            time.sleep(30)
+            if srv_restarted is False:
+                raise TestExecError("RGW service restart failed")
+            else:
+                log.info("RGW service restarted")
+            container_name = utils.gen_bucket_name_from_userid(
+                user_info["user_id"], rand_no=str(cc) + "_swift"
+            )
+            container = swiftlib.resource_op(
```

Comment:
lets move container creation method to reusable.py

Reply:
made changes
updated same log